src/__tests__/utils-test.ts

```diff
@@ -221,7 +222,9 @@ test('isABTestingAvailable happy case', async () => {
     });
 });
 
-test('isABTestingAvailable without bridge', async () => {
+// This test is not verifying anything. The remote config was set up by previous test
+// If you attempt to check value of key1 and key2 instead of key3 and key4 it fails
+test.skip('isABTestingAvailable without bridge', async () => {
```

Comment:
See comment, I think it's best to not leave the test having false security, because in actuallity it verifies no behaviour

Reply:
the remove it